tests/Tests/QuickCheckUtils.hs

```diff
@@ -49,7 +49,7 @@ import qualified Data.Text.Lazy as TL
 import qualified System.IO as IO
 
 genWord8 :: Gen Word8
-genWord8 = chooseAny
+genWord8 = choose (minBound, maxBound)
```

Comment:
I don't understand the commit message:  the bounds are `QuickCheck >= 2.14.1 && < 2.15`?

Reply:
The thing is that in emulated environment I have no resources to build a proper QuickCheck and must rely on whatever is `libghc-quickcheck2-dev` is. For Ubuntu 20.04 it is [2.12.6.1](https://packages.ubuntu.com/focal/libghc-quickcheck2-dev). Since `ghc --make` does not even look at cabal file, I can get away without changing bounds.

Let me do it in a proper way, with a CPP switch.

Reply:
Ah, right, I do not need CPP at all, just import `chooseAny` from `Test.QuickCheck.Gen` instead of `Test.QuickCheck`.

Reply:
Thanks, this is fixed now.